src/components/Suggestions/Suggestions.js

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+
+import Header from '../Header/Header'
+import MovieList from '../MovieList/MovieList'
+import Button from '../Button/Button'
+
+export default function Suggestions({ filteredMovies, borderHeight }) {
+  const [suggestions, setSuggestions] = useState(movieRandomizer())
+
+  return (
+    <div>
```

Comment:
In every page you have a `div` element around your elements. Maybe check whether you could use fragments `<>...</>` instead. It will depend on your layout. But would be nicer to not have a div in a div in a div... 😸